bin/pycbc_live

```diff
@@ -287,6 +285,54 @@ class LiveEventManager(object):
                 if store_psd[ifo] is not None:
                     store_psd[ifo].save(fname, group='%s/psd' % ifo)
 
+def init_data_reader(ifo, args, maxlen):
```

Comment:
Should this be a class method of StrainBuffer?

Reply:
Makes sense, moved to a `from_cli` class method.